### PR TITLE
Typo "a Teams Room"→"the Teams Rooms"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://learn.microsoft.com/en-us/microsoftteams/hybrid-meetings-device-config-calendar 
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/includes/mtr-device-config-calendar-include.md 
#PingMSFTDocs

### CODE REVIEW

File: Teams/includes/mtr-device-config-calendar-include.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 
-To help your users more easily schedule meetings in a Teams Room, you can create room lists and places in Exchange Online. 
+To help your users more easily schedule meetings in the Teams Rooms, you can create room lists and places in Exchange Online.
```
Comment (JohanFreelancer9) [inline Teams/includes/mtr-device-config-calendar-include.md:2]:
```suggestion
To help your users more easily schedule meetings in Teams Rooms, you can create room lists and places in Exchange Online. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by hyoshioka0128** [comment]
@JohanFreelancer9 - Thanks for the suggestion.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@tiburd Hope all is well. Could you please help with this issue? Thanks!

**Comment by tiburd** [comment]
This is not a typo. Closing this PR.
cc: @scanum

**Comment by dstrome** [comment]
Existing text is correct.

**Pull request closed without merging**